src/index.js

```diff
@@ -1,29 +1,75 @@
+/* eslint import/no-unresolved:0 */
 import './style.css';
+import addTodoItem from './modules/addTodoItem.js';
 
-const list = document.getElementById('lists');
-
-const data = [
-  {
-    index: 0,
-    description: 'Go for Shopping',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'Finish To Do List Project',
-    completed: false,
-  },
-];
+const getTodoTask = () => {
+  let todos;
+  if (localStorage.getItem('todos') === null) {
+    todos = [];
+  } else {
+    todos = JSON.parse(localStorage.getItem('todos'));
+  }
+  return todos;
+};
 
 const addTodoTask = (todo) => {
-  list.innerHTML += `<li id=${todo.index} class="listTask" completed=${false}> <div class="listTaskWrapper"><input type="checkbox" name="" id="checkbox"> <p class="desciption">${todo.description}<p/></div>
-    <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24"><path d="M12.025 21.15q-1 0-1.7-.7t-.7-1.675q0-1 .7-1.7t1.675-.7q1 0 1.688.7.687.7.687 1.7 0 .975-.687 1.675-.688.7-1.663.7Zm0-6.775q-1 0-1.7-.7T9.625 12q0-1 .7-1.688.7-.687 1.675-.687 1 0 1.688.687.687.688.687 1.663 0 1-.687 1.7-.688.7-1.663.7Zm0-6.75q-1 0-1.7-.713-.7-.712-.7-1.687 0-1 .7-1.688.7-.687 1.675-.687 1 0 1.688.687.687.688.687 1.688 0 .975-.687 1.687-.688.713-1.663.713Z" /></svg>
-    </li>
-    `;
+  const todos = getTodoTask();
+  todos.push(todo);
+  localStorage.setItem('todos', JSON.stringify(todos));
+};
+
+const updateTodoTask = (index, description) => {
+  const todos = getTodoTask();
+  const todo = todos.find((todoTask) => todoTask.index === index);
+  todo.description = description;
+  localStorage.setItem('todos', JSON.stringify(todos));
 };
 
 const display = () => {
-  data.map((todo) => addTodoTask(todo));
+  const todos = getTodoTask();
+  if (todos) {
+    todos.map((todo) => addTodoTask(todo));
+  }
 };
```

Comment:
- [ ] There's a problem with your `display` function, if you look closely, you're calling `addTodoTask` for each todo, but `addTodoTask` only pushes the todo into the `todos` array and then updates the local storage with the newly created todo, ending up with no effect in the DOM. 🤔 I think you confused `addTodoTask` with `addTodoItem` which is the one in charge of the UI. Please consider fixing this! 🤓 🔧